Mail/Views/New Message/ComposeMessageBodyView.swift

```diff
@@ -30,15 +47,109 @@ struct ComposeMessageBodyView: View {
 
     let messageReply: MessageReply?
 
+    private var isRemoteContentBlocked: Bool {
+        return UserDefaults.shared.displayExternalContent == .askMe && messageReply?.frozenMessage.localSafeDisplay == false
+    }
+
     var body: some View {
         VStack {
-            AttachmentsHeaderView()
-            ComposeEditor(
-                focusedField: _focusedField,
-                draft: draft,
-                isShowingAI: $isShowingAI,
-                messageReply: messageReply
+            #if os(macOS) || targetEnvironment(macCatalyst)
+            ComposeMessageMacosToolbarView(
```

Comment:
You can move this view in the same folder where is the iOS toolbar
You can also rename both toolbars, something like editorMobileToolbarView and editorDesktopToolbarView